.router.php

```diff
@@ -3,10 +3,12 @@
 
 $filename = $_SERVER["PATH_INFO"] ?? $_SERVER["SCRIPT_NAME"];
 
-if (file_exists($_SERVER["DOCUMENT_ROOT"] . $filename)) {
-    /* This could be an image or whatever, so don't try to compress it */
-    ini_set("zlib.output_compression", 0);
-    return false;
+if (!file_exists($_SERVER["DOCUMENT_ROOT"] . $filename)) {
+    require_once __DIR__ . '/error.php';
+
+    return;
 }
 
-include_once "error.php";
+/* This could be an image or whatever, so don't try to compress it */
+ini_set("zlib.output_compression", 0);
+return false;
```

Comment:
I would prefer:
```suggestion
return null;
```
Here and above.